allvars.h

```diff
@@ -520,7 +520,7 @@ extern struct particle_data
     pthread_spinlock_t SpinLock;
 #endif
 
-    float GravCost;     /*!< weight factor used for balancing the work-load */
+    int GravCost;     /*!< weight factor used for balancing the work-load */
```

Comment:
Second thought when this rounds to zero -- it means GravCost is dominated by the gravity isn't that actually fair? This way All.HydroCostFactor roughly sets the redshift where we start to favour balancing by Hydro.